plugins/environment.go

```diff
@@ -115,11 +116,17 @@ When the -plugin option is specified, these flags are ignored.`)
 		err = proto.Unmarshal(apiData, documentv3)
 		if err == nil {
 			env.Request.AddModel("openapi.v3.Document", documentv3)
+			sourceName := strings.ReplaceAll(*input, ".pb", ".yaml")
```

Comment:
`strings.ReplaceAll` is a `go 1.12` API, it's causing CI to fail, which  is using `go 1.11`. It's reasonable to support `1.11` so let's use `strings.Replace(*input, ".pb", ".yaml", -1)`.

Reply:
Done